Makefile

```diff
@@ -103,8 +103,8 @@ OCI_REGISTRY := projects.registry.vmware.com/tce
 ##### IMAGE #####
 
 ##### LINTING TARGETS #####
-.PHONY: lint mdlint shellcheck check yamllint misspell actionlint urllint imagelint
-check: ensure-deps lint mdlint shellcheck yamllint misspell actionlint urllint imagelint
+.PHONY: lint mdlint shellcheck check yamllint misspell actionlint urllint imagelint e2e-test
+check: ensure-deps lint mdlint shellcheck yamllint misspell actionlint urllint imagelint e2e-test
```

Comment:
@aman556 There's no `e2e-test` target in the `Makefile` but I see it referenced here. This will fail if someone runs `make check`

Reply:
Thanks @karuppiah7890 I will do this change in a while.